src/index.js

```
@@ -1 +1,16 @@
 'use strict';
+require('dotenv').config({ path: '../.env' });
+require('./models/user');
+
+const { createServer } = require('./createServer');
+const { sequelize } = require('./utils/db');
+
+(async () => {
+  try {
+    await sequelize.authenticate();
+
+    await sequelize.sync({ alter: true });
+
+    createServer().listen(5700, () => {});
+  } catch (error) {}
```

The `catch (error) {}` block silently swallows any errors during DB authentication/sync or server start. This will make debugging startup failures very difficult. At minimum log the error (`console.error(error)`) and exit the process (`process.exit(1)`) or rethrow so the failure is visible in CI/hosting logs.